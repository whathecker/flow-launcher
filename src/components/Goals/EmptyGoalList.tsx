import React from "react";
import { Image, StyleSheet } from "react-native";
import { View, Text } from "../Themed";
import { Typography, Color } from "../../styles";

const EmptyGoalList: React.FC = () => {
  const darkTextColor = Color.light.subtleLabel;
  const lightTextColor = Color.light.subtleLabel;
  return (
    <>
      <View style={styles.blockWrapper}>
        <Image
          style={styles.image}
          source={require(`../../../assets/images/sauropod_1f995.png`)}
        />
      </View>
      <View style={styles.blockWrapper}>
        <Text
          lightColor={lightTextColor}
          darkColor={darkTextColor}
          style={styles.text}
        >{`The journey of a thousand miles`}</Text>
        <Text
          lightColor={lightTextColor}
          darkColor={darkTextColor}
          style={styles.text}
        >{`begins with a single step.`}</Text>
      </View>
      <View style={styles.blockWrapper}>
        <Text
          lightColor={lightTextColor}
          darkColor={darkTextColor}
          style={styles.text}
        >{`LAO TZU`}</Text>
      </View>
    </>
  );
};

const styles = StyleSheet.create({
  blockWrapper: {
    paddingTop: 5,
    paddingBottom: 18,
  },
  image: {
    width: 95,
    height: 95,
  },
  text: {
    ...Typography.p,
    textAlign: "center",
  },
});

export default EmptyGoalList;
