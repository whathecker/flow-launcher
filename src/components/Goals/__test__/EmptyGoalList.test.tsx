import * as React from "react";
import renderer from "react-test-renderer";
import EmptyGoalList from "../EmptyGoalList";

describe("Test EmptyGoalList component", () => {
  it("Should renders EmptyGoalList correctly", () => {
    const tree = renderer.create(<EmptyGoalList />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
