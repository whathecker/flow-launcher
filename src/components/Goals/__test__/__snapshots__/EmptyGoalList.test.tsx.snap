// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test EmptyGoalList component Should renders EmptyGoalList correctly 1`] = `
Array [
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#fff",
        },
        Object {
          "paddingBottom": 18,
          "paddingTop": 5,
        },
      ]
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 95,
          "width": 95,
        }
      }
    />
  </View>,
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#fff",
        },
        Object {
          "paddingBottom": 18,
          "paddingTop": 5,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
          },
          Object {
            "fontFamily": "Lexend-Regular",
            "fontSize": 18,
            "textAlign": "center",
          },
        ]
      }
    >
      The journey of a thousand miles
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
          },
          Object {
            "fontFamily": "Lexend-Regular",
            "fontSize": 18,
            "textAlign": "center",
          },
        ]
      }
    >
      begins with a single step.
    </Text>
  </View>,
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#fff",
        },
        Object {
          "paddingBottom": 18,
          "paddingTop": 5,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
          },
          Object {
            "fontFamily": "Lexend-Regular",
            "fontSize": 18,
            "textAlign": "center",
          },
        ]
      }
    >
      LAO TZU
    </Text>
  </View>,
]
`;
